pandas/conftest.py

```diff
@@ -675,3 +675,20 @@ def tick_classes(request):
         normalize=st.booleans(),
         startingMonth=st.integers(min_value=1, max_value=12)
     ))
+
+lorem_ipsum_text = (
```

Comment:
Can just define this in the function

Reply:
Or just declare it in test_to_html. That seems easier to follow.